library/examples/hate_speech_meta_sklearn.py

```diff
@@ -0,0 +1,156 @@
+from copy import deepcopy
+
+
+from blocks.adaptors import ListOfListsToNumpy
+from blocks.augmenters.spelling_autocorrect import SpellAutocorrectAugmenter
+from blocks.concat import DataSource, VectorConcat
+
+from blocks.ensemble import Ensemble
+from blocks.models.huggingface import HuggingfaceModel
+from blocks.pipeline import Pipeline
+
+from blocks.transformations import (
+    Lemmatizer,
+    SKLearnTransformation,
+    SpacyTokenizer,
+    TextStatisticTransformation,
+)
+from configs.constants import Const
+from data.transformation import transform_dataset
+from datasets.load import load_dataset
+from library.evaluation import classification_metrics  # , calibration_metrics
+from sklearn.ensemble import GradientBoostingClassifier, VotingClassifier
+from sklearn.feature_extraction.text import TfidfVectorizer
+from sklearn.linear_model import LogisticRegression
+from sklearn.naive_bayes import MultinomialNB
+from sklearn.preprocessing import MinMaxScaler
+from transformers import TrainingArguments
+from type import (
+    DatasetSplit,
+    HuggingfaceConfig,
+    LoadOrigin,
+    PreprocessConfig,
+    Experiment,
+    SKLearnConfig,
+    HFTaskTypes,
+)
+from blocks.models.sklearn import SKLearnModel
+from blocks.adaptors.classification_output import ClassificationOutputAdaptor
+
+from utils.flatten import remove_none
+from data.dataloader import DataLoader
+
+""" Models """
+preprocess_config = PreprocessConfig(
+    train_size=-1,
+    val_size=-1,
+    test_size=-1,
+    input_col="text",
+    label_col="label",
+)
+
+
+sklearn_config = SKLearnConfig(
```

Comment:
you could import this from the other file if you wanted to, I tried  to make these less redundant, but it's not a must - we can deal with that later aswell

Reply:
Makes sense! I'll do it at a later PR.